packages/uswds-core/src/styles/mixins/general/add-aspect.scss

```diff
@@ -2,21 +2,56 @@
 @use "sass:map";
 @use "../../functions" as *;
 @use "../../variables" as *;
+@use "../helpers/at-media.scss" as *;
 
-@mixin add-aspect($ratio...) {
-  $ratio: unpack($ratio);
-  $important: null;
-  @if has-important($ratio) {
-    $ratio: list.nth($ratio, 1);
-    $important: "!important";
+@mixin add-aspect-children{
+  & > * {
+    position: absolute;
+    top: 0;
+    left: 0;
+    width: 100%;
+    height: 100%;
+  }
+
+  @at-root #{selector-append(img, &)},
+  & > img {
+    object-fit: cover;
   }
```

Comment:
```
.test-add-aspect {
    @include add-aspect("1x1");
}
```
Will output:
```
img.test-add-aspect, .test-add-aspect > img{
    object-fit:cover;
 }
 ```
It is the Sass alternative for `img&`, which Sass does not currently support. Open Sass issue explaining the issue [here](https://github.com/sass/sass/issues/1425). 

I did discover this can be accomplished with a slightly less verbose solution (`@at-root img#{&}`) and just pushed that up.